tests/dragonfly/connection_test.py

```diff
@@ -1080,7 +1080,7 @@ async def test_timeout(df_server: DflyInstance, async_client: aioredis.Redis):
     assert int(info["timeout_disconnects"]) >= 1
 
 
-@dfly_args({"send_timeout": 3})
+@dfly_args({"send_timeout": 5})
```

Comment:
I dont think this is the timeout that needs to be fixed.
The test failed due to 
wait_for_conn_drop fail, right? if so we need to increase the timeout we have in wait_for_conn_drop that we wait for the condition to meet